packages/js/product-editor/src/components/attribute-input-field/menu-attribute-list.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { plus } from '@wordpress/icons';
+import { Icon } from '@wordpress/components';
+import { createElement } from '@wordpress/element';
+import { __experimentalSelectControlMenuItem as MenuItem } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	MenuAttributeListProps,
+	NarrowedQueryAttribute,
+	UseComboboxGetMenuPropsOptions,
+} from './types';
+
+function isNewAttributeListItem( attribute: NarrowedQueryAttribute ): boolean {
+	return attribute.id === -99;
+}
+
+function sanitizeSlugName( slug: string | undefined ): string {
+	return slug && slug.startsWith( 'pa_' ) ? slug.substring( 3 ) : '';
+}
+
+export const MenuAttributeList: React.FC< MenuAttributeListProps > = ( {
+	disabledAttributeMessage = '',
+	renderItems,
+	highlightedIndex,
+	getItemProps,
+} ) => {
+	if ( renderItems.length > 0 ) {
+		return (
+			<div>
```

Comment:
Consider to replace this element with a `<Fragment />` element. Otherwise, it will add an unneeded `<div />` Dom element.